packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -2260,4 +2266,26 @@ describe('meeting', () => {
       });
     });
   });
+  describe('utility', () => {
```

Comment:
Please split these into separate tests (one test for each invalid case, and then a single test for all of the valid ones together is probably okay) This will make it more obvious what is wrong if something breaks.

Reply:
Gotcha - I split it into two separate files